src/ucp/core/ucp_context.c

```diff
@@ -1549,6 +1562,7 @@ static void ucp_free_config(ucp_context_h context)
 {
     ucs_free(context->config.alloc_methods);
     ucs_free(context->config.env_prefix);
+    ucs_free(context->config.ext.selection_cmp);
```

Comment:
reverse order of free between env_prefix and selection_cmp

Reply:
I reversed the order of allocation instead.